.github/workflows/check_links.yml

```diff
@@ -0,0 +1,15 @@
+name: check_links
+
+on: [push, pull_request]
+
+jobs:
+  check-links:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+    - uses: actions/checkout@v1
+    - name: Link Checker
+      id: lc
+      uses: peter-evans/link-checker@v1
```

Comment:
It appears that this particular action is now deprecated - should we use the suggested one - https://github.com/peter-evans/link-checker, https://github.com/lycheeverse/lychee-action ?

Reply:
It seems to still work, even if it's not being maintained anymore. We could fork the action to a private repo (since we use it in another repo anyway), and continue using it instead of switching.

Reply:
Attempted to switch to lychee-action, but it came with reduced functionality. The current version does not appear to support relative links to files in GitHub, and would only be able to check external links.

Reply:
Per our conversation, the other package has reduced functionality for relative links, so continuing with the first one